Sources/Sonar/APIs/AppleRadarRouter.swift

```diff
@@ -59,7 +60,7 @@ enum AppleRadarRouter {
                     "configurationSplitValue": radar.configuration,
                     "workAroundText": "",
                     "descriptionText": radar.body,
-                    "problemAreaTypeCode": radar.area.map { String($0.appleIdentifier) } ?? "",
+                    "problemAreaTypeCode": (radar.area.map { String($0.appleIdentifier) } ?? "") as Any,
```

Comment:
This has to be a bug, if I take this statement out of the dictionary literal it is correctly typed.